sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -66,6 +66,7 @@ class InteractiveRunnerTest(unittest.TestCase):
   def setUp(self):
     ie.new_env()
 
+  @unittest.skipIf(sys.platform == "win32", "[BEAM-10627]")
```

Comment:
@KevinGG - assigned this jira to you.

Reply:
Acked. It's a duplicate of BEAM-10514. The reason is the file system in test has a temp directory with a very long prefix, causing the file path exceeding windows default 260 character limit.

A normal windows temp directory prefix: `c:\windows\temp\`, 
In the test: `D:\a\beam\beam\sdks\python\target\.tox\py35-win\tmp\`.

The length diff breaks the limit since usually Interactive Beam uses about 240 characters in path length.

Reply:
Thanks @KevinGG . Should I put `BEAM-10514` here or keep `BEAM-10627`?